lib/components/user/user-account-screen.js

```diff
@@ -107,35 +109,10 @@ class UserAccountScreen extends Component {
   // TODO: Update title bar during componentDidMount.
 
   render () {
-    const { auth0, loggedInUser } = this.props
-    const { userData } = this.state
-
-    let formContents
-    if (isNewUser(loggedInUser)) {
-      if (!auth0.user.email_verified) {
-        // Check and prompt for email verification first to avoid extra user wait.
-        formContents = <VerifyEmailScreen />
-      } else {
-        // New users are shown "wizard" (step-by-step) mode
-        // (includes when a "new" user clicks 'My Account' from the account menu in the nav bar).
-        formContents = (
-          <NewAccountWizard
-            onComplete={this._handleExitAndSave}
-            panes={this._panes}
-            userData={userData}
-          />
-        )
-      }
-    } else {
-      formContents = (
-        // Existing users are shown all panes together.
-        <ExistingAccountDisplay
-          onCancel={this._handleExit}
-          onComplete={this._handleExitAndSave}
-          panes={this._panes}
-        />
-      )
-    }
+    const { isEmailVerified, loggedInUser } = this.props
+
+    // Capture the handler for use in the Formik callback (where `this` might mean something else.)
+    const handleExit = this._handleExit
```

Comment:
I'm not sure why this is treated differently than passing `this._panes` or `this._handleSaveAndExit`.

Reply:
I thought `this` inside the Formik component could have another meaning but it turns out not to be the case. This is handled by commit https://github.com/opentripplanner/otp-react-redux/commit/6902c2b4c80245cb0821fdf0b69ec5406dbb242f.